service/src/main/java/io/camunda/service/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -94,8 +93,8 @@ private SearchQuery getProcessInstanceKeysQuery(final List<Long> processInstance
     return longTerms("processInstanceId", processInstanceKeys);
   }
 
-  private SearchQuery getProcessDefinitionKeyQuery(final List<String> processDefinitionIds) {
-    return stringTerms("processDefinitionId", processDefinitionIds);
+  private SearchQuery getProcessDefinitionKeyQuery(final List<Long> processDefinitionIds) {
+    return longTerms("processDefinitionId", processDefinitionIds);
```

Comment:
❓ Just to understand this: on the Tasklist index, process definition key (the Long identifier internally generated by Zeebe) is named `processDefinitionId`, that is why we're mapping it here accordingly?

Reply:
Yes, you are right over here 
![image](https://github.com/user-attachments/assets/5cecc960-faee-4884-a2c1-d3f17e1ba338)